tests/unit/test_rat_board.py

```diff
@@ -8,6 +8,9 @@
 
 from src.packages.board.board import cycle_at
 
+import datetime
+from datetime import timezone
```

Comment:
[suggestion] this can be achieved in a single import, and would remove a level of ambiguity.

something to the effect of 
```py
from datetime import datetime, timezone
```

Reply:
Updated, also had to update case_management.py in more commands to make sure it all kept working (datetime.datetime -> datetime)